src/marketing/prex-type-morelikethis.md

```diff
@@ -0,0 +1,25 @@
+---
+ee_only: true
+title: More like this
+---
+
+The _More like this_ recommendation type recommends items based on similar content and attributes to the product being viewed.
+
+## How it works
+
+By evaluating the attributes for the products being viewed or in a cart or on the confirmation page, _More like this_ recommends other similar products within those product categories. For example, if a shopper is browsing yoga mats, the _More like this_ recommendation type recommends other products within the equipment category. Because _More like this_ does not currently distinguish between genders, this recommendation type is not recommended for clothing stores.
```

Comment:
>  products being viewed or in a cart or on the confirmation page

I would simplify - no need to list out all possibilities here 
> those product categories

a little unclear which categories you're talking about. could link to the existing filtering topic 

> not recommended for clothing stores.

I would say "apparel, fashion, or other gender-specific verticals"